index.bs

```diff
@@ -68,6 +68,9 @@ spec: FETCH; urlPrefix: https://fetch.spec.whatwg.org/
 spec: promises-guide-1; urlPrefix: https://www.w3.org/2001/tag/doc/promises-guide
   type: dfn
     text: promise-calling; url: should-promise-call
+spec: infra; urlPrefix: https://infra.spec.whatwg.org/
+  type: dfn
+    text: exists; url: map-exists
```

Comment:
https://infra.spec.whatwg.org/#map-exists is exported, so you don't need to define it locally if you use the right `[=for/=]` syntax below.

```suggestion
```